rpm/quota.spec

```diff
@@ -116,6 +117,9 @@ install -p -m644 -D %{SOURCE1} %{buildroot}/%{_unitdir}/quota_nld.service
 install -p -d %{buildroot}/%{_unitdir}/multi-user.target.wants/
 ln -s ../quota_nld.service %{buildroot}/%{_unitdir}/multi-user.target.wants/
 
+mkdir -p %{buildroot}/%{_unitdir}/quota@home.service.d
+install -p -m644 -D %{SOURCE2} %{buildroot}/%{_unitdir}/quota@home.service.d/
```

Comment:
It seems that both `quota@home.service.d` and `quota@.service.d` work, and I'd assume this has the effect that former runs only when for `home` and latter is more generic for `any` mount. Should we keep it as specific here?